dev/source/docs/how-the-team-works-voting.rst

```diff
@@ -42,9 +42,9 @@ Description belows how to creat a form from scratch, alternatively you can use `
   - set the "Email To" field to "ardupilot-dev-team@googlegroups.com"
   - fill in the Subject and Message fields
   - to allow others to see the responses (including ardupilot.devel@gmail.com), click the "Collaborators" link and add their email in the "Invite people" field.  If you are submitting the vote as yourself, include 
-  - check "Include form in email" and press "Send"
+  - check "Include form in email" and press "Send".
```

Comment:
Not really necessary, but the line above it really does not make sense....I know its existing, but @rmackay9 , what are you supposed to include? if its a vote for yourself?

Reply:
@rmackay9 bump

Reply:
I think we should zap the end of the line, "If you are submitting the vote as yourself, include "

Reply:
zapped....by a direct commit